src/chainlit/server.py

```diff
@@ -180,7 +180,7 @@ def get_html_template():
         content = f.read()
         content = content.replace(PLACEHOLDER, tags)
         if js:
-            content = content.replace(JS_PLACEHOLDER, js)
+            content = content.replace(JS_PLACEHOLDER, js.replace("None", "undefined"))
```

Comment:
this is not enough as True and False will also be problematic. can we try something like

```py
js = f"""<script>window.theme = {json.dumps(config.ui.theme.to_dict())}</script>"""
```

Reply:
Looks better!